.github/workflows/cria-executaveis.yml

```diff
@@ -54,16 +55,16 @@ jobs:
         with:
           name: corretor-pyinstaller-artifact
 
-      - name: Zip file windows
+      - name: Zippar arquivos Windows
         run: zip -r corretor_windows.zip dist-windows-*
 
-      - name: Zip file ubuntu
+      - name: Zippar arquivos Ubuntu
         run: zip -r corretor_ubuntu.zip dist-ubuntu-*
 
```

Comment:
@vianaz Uma sugestão, aqui talvez da pra tirar o zip de ubuntu, mas se não for fazer mal acho que não tem problema deixar também.

Reply:
@maecco feito!